cumulusci/tasks/push/tasks.py

```diff
@@ -236,7 +240,7 @@ def _init_task(self):
 
     def _init_options(self, kwargs):
         super(SchedulePushOrgList, self)._init_options(kwargs)
-
+        self.options["dry_run"] = process_bool_arg(self.options.get("dry_run", False))
```

Comment:
I think the dry_run option is valuable for SchedulePushOrgQuery, in order to have a way to check how many orgs were selected. But I don't see the value for SchedulePushOrgList, where the user is providing the list. I would just set it to False here and not expose the option.